roles/lighthouse/defaults/main.yaml

```diff
@@ -0,0 +1,47 @@
+lighthouse_user: lighthouse
+lighthouse_datadir: /data/lighthouse
+lighthouse_auth_jwt_path: /data/execution-auth.secret
+lighthouse_execution_engine_endpoint: http://geth:8551
+
+lighthouse_cleanup: false # when set to "true" it will remove the container
+
+lighthouse_ports_p2p_tcp: 9000
+lighthouse_ports_p2p_udp: 9000
+lighthouse_ports_http_beacon: 5052
+lighthouse_ports_metrics: 5054
+
+lighthouse_container_name: lighthouse
+lighthouse_container_image: sigp/lighthouse:latest
+lighthouse_container_env: {}
+lighthouse_container_ports:
+  - "127.0.0.1:{{ lighthouse_ports_http_beacon }}:{{ lighthouse_ports_http_beacon }}"
+  - "{{ lighthouse_ports_p2p_tcp }}:{{ lighthouse_ports_p2p_tcp }}"
+  - "{{ lighthouse_ports_p2p_udp }}:{{ lighthouse_ports_p2p_udp }}/udp"
+lighthouse_container_volumes:
+  - "{{ lighthouse_datadir }}:/data"
+  - "{{ lighthouse_auth_jwt_path }}:/execution-auth.jwt:ro"
+lighthouse_container_stop_timeout: "300"
+lighthouse_container_networks: []
+lighthouse_container_command:
+  - lighthouse
+  - beacon_node
+  - --datadir=/data
+  - --disable-upnp
+  - --disable-enr-auto-update
+  - --enr-address={{ ansible_host }}
+  - --enr-tcp-port={{ lighthouse_ports_p2p_tcp }}
+  - --enr-udp-port={{ lighthouse_ports_p2p_udp }}
+  - --listen-address=0.0.0.0
+  - --port={{ lighthouse_ports_p2p_tcp }}
+  - --discovery-port={{ lighthouse_ports_p2p_udp }}
+  - --http
+  - --http-address=0.0.0.0
+  - --http-port={{ lighthouse_ports_http_beacon}}
+  - --execution-jwt=/execution-auth.jwt
+  - --execution-endpoint={{ lighthouse_execution_engine_endpoint }}
```

Comment:
This approach would imply that we no longer have the `snooper` container, that's the mitm proxy we have dumping engine API requests. I'd say that should be enabled/disabled via a flag since its an invaluable debug tool on testnets. The current approach is to basically do this:
- EL started with `--authrpc.port={{el_engine_port}}`
- EL sets value for variable `snoop_command: "./json_rpc_snoop -p {{el_engine_snoop_port}} http://127.0.0.1:{{el_engine_port}}"` and `execution_endpoint: http://127.0.0.1:{{el_engine_snoop_port}}`. This allows us to pick and choose how each EL handles snooping ports. 
- The CL then sends requests via `--execution-endpoint={{execution_endpoint}}`, which would be sending requests to the `el_engine_snoop_port`, i.e `snooper` container

I guess some variation of this logic needs to be available here, perhaps with a flag since its a feature in the role and not a requirement.

Reply:
That's fine, we can run a snooper container and overwrite the default command specifically for our testnet environments to interact with the snooper :+1:  It doesn't need to be part of the client roles, it is specific to our testnets.

Reply:
Yup, will wait for a example on a testnet to check if it behaves the same :D